src/Microsoft.OpenApi/Extensions/StringExtensions.cs

```diff
@@ -30,19 +30,19 @@ internal static class StringExtensions
             return false;
 
         }
-        internal static bool TryGetEnumFromDisplayName<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] T>(this string displayName, out T result) where T : Enum
+        internal static bool TryGetEnumFromDisplayName<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] T>(this string? displayName, out T result) where T : Enum
         {
             var type = typeof(T);
 
-            var displayMap = EnumDisplayCache.GetOrAdd(type, _=> GetEnumValues<T>(type));
+            var displayMap = EnumDisplayCache.GetOrAdd(type, _ => GetEnumValues<T>(type));
 
-            if (displayMap.TryGetValue(displayName, out var cachedValue))
+            if (displayName is not null && displayMap.TryGetValue(displayName, out var cachedValue))
             {
                 result = (T)cachedValue;
                 return true;
             }
 
-            result = default;
+            result = default!;
```

Comment:
avoid using bang operators, make result nullable instead and add a "NotNullWhen(true)" attribute

Reply:
The `NotNullWhen(true)` attribute is inaccessible since its an internal sealed class but I'll make the result nullable